### ISSUE TITLE
在huggingface.co的chinese-xlnet-mid预训练模型做生成任务，没有结果

### ISSUE THREAD

**Issue by ljq2278** [ymcui/Chinese-XLNet#60]
https://huggingface.co/hfl/chinese-xlnet-mid里做的文本生成任务，为什么没有结果呢？另外按照https://huggingface.co/transformers/model_doc/xlnet.html官网例子

import tensorflow as tf
import numpy as np
from transformers import XLNetTokenizer, TFXLNetLMHeadModel
tokenizer = XLNetTokenizer.from_pretrained('hfl/chinese-xlnet-mid')
model = TFXLNetLMHeadModel.from_pretrained('hfl/chinese-xlnet-mid')

input_ids = tf.constant(tokenizer.encode("我的狗很\<mask\>", add_special_tokens=True))[None, :]  # We will predict the masked token
perm_mask = np.zeros((1, input_ids.shape[1], input_ids.shape[1]))
perm_mask[:, :, -1] = 1.0  # Previous tokens don't see last token
target_mapping = np.zeros((1, 1, input_ids.shape[1]))  # Shape [1, 1, seq_length] => let's predict one token
target_mapping[0, 0, -1] = 1.0  # Our first (and only) prediction will be the last token of the sequence (the masked token)
outputs = model(input_ids, perm_mask=tf.constant(perm_mask, dtype=tf.float32), target_mapping=tf.constant(target_mapping, dtype=tf.float32))
next_token_logits = outputs[0]  # Output has shape [target_mapping.size(0), target_mapping.size(1), config.vocab_size]

预测的东西不对劲，啥原因呢
